File: 1440-convert-integer-to-the-sum-of-two-no-zero-integers/convert-integer-to-the-sum-of-two-no-zero-integers.js
/**
 * @param {number} n
 * @return {number[]}
 */
var getNoZeroIntegers = function(n) {
    let a = 1
    let b = n-1
    while (true){
        let a1 = a.toString()
        let b1 = b.toString()
        if(!a1.includes("0") && !b1.includes("0")){
            return [a, b]
        }
        a+=1
        b-=1
    }
};

File: 1440-convert-integer-to-the-sum-of-two-no-zero-integers/convert-integer-to-the-sum-of-two-no-zero-integers.py
class Solution:
    def getNoZeroIntegers(self, n: int) -> List[int]:
        a = 0
        b = n
        while True:
            a1 = str(a)
            b1 = str(b)
            if "0" in a1 or "0" in b1:
                a+=1
                b-=1
            if a+b == n and "0" not in a1 and "0" not in b1:
                break
        return [a,b]